### PR TITLE
readds fbp whitelist and the things assorted with it 

### PR DESCRIPTION
**Pull request by breadcrombs**
## About the Pull Request

Readds @Waklaidan's fbp whitelist with some fixes

## Why It's Good For The Game

Re adds the long awaited fbp whitelist that got nuked

## Changelog

:cl:
add: Option for fbp again (you need fbp wl or you can't spawn)
add: FBP whitelist
/:cl:

### CODE REVIEW

**Comment by breadcrombs** [comment]
Some explanation: (I tested it)  Trying to give someone wl just by ckey - SPECIES_IPC or ckey- SPECIES_FBP dosen't work, you must do ckey - All. (dosen't hurt since all other races are disabled).

**Comment by Calyxman** [comment]
speedmerge pls

**Comment by breadcrombs** [comment]
unless they are using the sql database, everytime the server gets updated they need to refill the alien whitelist similar to the admin one

**Comment by Waklaidan** [comment]
> Some explanation: (I tested it) Trying to give someone wl just by ckey - SPECIES_IPC or ckey- SPECIES_FBP dosen't work, you must do ckey - All. (dosen't hurt since all other races are disabled).

It worked for me. I just entered ckey - SPECIES_FBP. Maybe there is something wrong with your config.

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -254,6 +259,11 @@
 
 /mob/new_player/proc/create_character(turf/spawn_turf)
 	spawning = TRUE
+	if(client.prefs.organ_data[BP_CHEST] == "cyborg")
+		if(!whitelist_lookup(SPECIES_IPC, client.ckey) && client.prefs.species != SPECIES_IPC)
```
Comment (Waklaidan) [inline code/modules/mob/new_player/new_player.dm]: Replace SPECIES_IPC to SPECIES_FBP and define with

**Review by Waklaidan — CHANGES_REQUESTED** [review]

File: code/modules/mob/new_player/new_player.dm (hunk 1 above)
Comment (breadcrombs) [reply code/modules/mob/new_player/new_player.dm]: ok, I should test it again

**Comment by breadcrombs** [comment]
ok it worked for me, must be a configs folder error

**Pull request merged**